scylla/src/transport/retry_policy.rs

```diff
@@ -25,7 +25,7 @@ pub enum RetryDecision {
 }
 
 /// Specifies a policy used to decide when to retry a query
-pub trait RetryPolicy: Send + Sync {
+pub trait RetryPolicy: std::fmt::Debug + Send + Sync {
```

Comment:
This is a breaking API change, we will have to release a new version.

Reply:
yup, we will soon anyway